documentation/book/con-scaling-kafka-clusters.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// assembly-scaling-clusters.adoc
+
+[id='con-scaling-kafka-clusters-{context}']
+
+= Scaling Kafka clusters
+
+== Adding brokers to a cluster
+
+The primary way of increasing throughput for a topic is to increase the number of partitions for that topic. 
+That works because the partitions allow the load for that topic to be shared between the brokers in the cluster.
+When the brokers are all constrained by some resource (typically I/O), then using more partitions will not yield an increase in throughput.
+Instead, you must add brokers to the cluster.
```

Comment:
There's a difference between the imperative  "add more brokers" and the more descriptive "you need to ...", and in this case I think the descriptive form is better because it is explaining why adding brokers is necessary. Maybe it's just me...?

Reply:
Happy to add "you need to".